modules/iam-group/main.tf

```diff
@@ -31,7 +31,7 @@ resource "oci_identity_user_group_membership" "this" {
   # https://github.com/hashicorp/terraform/issues/12570
   # count    = "${length(var.user_ids)}"
   # use workaround here
-  count = "${var.user_count}"
+  count = "${var.group_create || length(lookup(local.group_ids[0], "id")) > 0 ? var.user_count : 0}"
```

Comment:
If group_create == false, it will check length(lookup(local.group_ids[0], "id") to see if this group already exist, if yes, it will add user to an existing group; if no, it will not create  oci_identity_user_group_membership.

Reply:
Ah, right, I misread that